editor/editor_dock_manager.h

```diff
@@ -70,49 +74,51 @@ class EditorDockManager : public Object {
 	};
 
 private:
+	friend class DockContextPopup;
+
+	struct DockInfo {
+		String title;
+		bool open = false;
+		bool enabled = true;
+		bool at_bottom = false;
+		int previous_tab_index = -1;
+		bool previous_at_bottom = false;
+		WindowWrapper *floating_window = nullptr;
+		int dock_slot_index = DOCK_SLOT_LEFT_UL;
+		Ref<Shortcut> shortcut;
+	};
+
 	static EditorDockManager *singleton;
 
 	// To access splits easily by index.
 	Vector<DockSplitContainer *> vsplits;
 	Vector<DockSplitContainer *> hsplits;
 
-	Vector<WindowWrapper *> floating_docks;
-	Vector<Control *> bottom_docks;
+	Vector<WindowWrapper *> floating_windows;
```

Comment:
Why was this renamed? Windows are already "floating", the previous name was more precise. If anything, it could be `dock_windows`.

Reply:
I changed it since it holds the Windows, not the docks themselves.
I'll change it to `dock_windows`, since that is more accurate.